website/client/mixins/spells.js

```diff
@@ -129,6 +129,28 @@ export default {
 
 
       this.markdown(msg); // @TODO: mardown directive?
+
+      // If using mpheal and there are other mages in the party, show extra notification
+      if (type === 'party' && spell.key === 'mpheal') {
+        // Get party
+        let members = await this.$store.dispatch('members:getGroupMembers', {
```

Comment:
why are we loading this data? isn't it already loaded since it's the user's own party?

Reply:
Is it? Can you tell me the variable name?
Sorry, I really looked for it, but didn't find.

Reply:
Now I understand why I wasn't finding this variable. There was a mistake on the code that passes the party members to the `castEnd` method.
`this.$store.state.partyMembers` returns an object, but after it there was an check it was an Array.
```javascript
let party = this.$store.state.partyMembers;
party = isArray(party) ? party : [];
party = party.concat(this.user);
this.castEnd(party, spell.target);
```
`party` was always an array with `this.user`.

I fixed it changing this code to:
```javascript
let party = this.$store.state.partyMembers.data;
if (!isArray(party)) {
  party = [this.user];
}
this.castEnd(party, spell.target);
```